scripts/lib/CIME/case/case_setup.py

```diff
@@ -152,10 +152,15 @@ def _case_setup_impl(case, caseroot, clean=False, test_mode=False, reset=False):
 
             case.initialize_derived_attributes()
 
-            cost_per_node = 16 if case.get_value("MACH") == "yellowstone" else case.get_value("MAX_MPITASKS_PER_NODE")
+            cost_per_node = case.get_value("MAX_MPITASKS_PER_NODE")
             case.set_value("COST_PES", case.num_nodes * cost_per_node)
```

Comment:
I see that there is COSTPES_PER_NODE variable in env_mach_pes.xml. Should these 2 lines be
```
             tasks_per_node = case.get_value("MAX_MPITASKS_PER_NODE")
             case.set_value("COST_PES", case.num_nodes * case.get_value("COSTPES_PER_NODE"))
```
because it looks like COSTPES_PER_NODE was specifically introduced to calculate COST_PES.

Reply:
Good point thanks, COSTPES_PER_NODE should be used.